Problem compiling with zig 0.9.0-dev.755+799fedf61

Hi, I seem to have an issue building zls with zig 0.9.0-dev.755+799fedf61

```
./src/setup.zig:228:31: error: expected type 'type', found '[]u8'
    var it = std.mem.tokenize(env_path, &[_]u8{std.fs.path.delimiter});
                              ^
./src/setup.zig:68:24: note: referenced here
    var zig_exe_path = try findZig(allocator);
                       ^
./src/main.zig:1805:13: note: referenced here
            try setup.wizard(allocator);
            ^
./src/known-folders/known-folders.zig:149:38: error: expected type 'type', found '*[8192]u8'
        var line_it = std.mem.split(&read, "\n");
                                     ^
zls...The following command exited with error code 1:
/home/build/zig/build/zig build-exe /home/build/zls/src/main.zig --pkg-begin build_options /home/build/zls/zig-cache/zls_build_options.zig --pkg-end --cache-dir /home/build/zls/zig-cache --global-cache-dir /home/build/.cache/zig --name zls --pkg-begin known-folders /home/build/zls/src/known-folders/known-folders.zig --pkg-end --enable-cache
error: the following build command failed with exit code 1:
/home/build/zls/zig-cache/o/0d41ce75e2baa15549c0a336c4e164e9/build /home/build/zig/build/zig /home/build/zls /home/build/zls/zig-cache /home/build/.cache/zig
```

--- a/src/document_store.zig
+++ b/src/document_store.zig
@@ -148,7 +148,7 @@ fn loadPackages(context: LoadPackagesContext) !void {
                 }
 
                 build_file.packages.shrinkAndFree(allocator, 0);
-                var line_it = std.mem.split(zig_run_result.stdout, "\n");
+                var line_it = std.mem.split(u8, zig_run_result.stdout, "\n");
                 while (line_it.next()) |line| {
                     if (std.mem.indexOfScalar(u8, line, '\x00')) |zero_byte_idx| {
                         const name = line[0..zero_byte_idx];

--- a/src/offsets.zig
+++ b/src/offsets.zig
@@ -14,7 +14,7 @@ pub const DocumentPosition = struct {
 };
 
 pub fn documentPosition(doc: types.TextDocument, position: types.Position, encoding: Encoding) !DocumentPosition {
-    var split_iterator = std.mem.split(doc.text, "\n");
+    var split_iterator = std.mem.split(u8, doc.text, "\n");
 
     var line_idx: i64 = 0;
     var line: []const u8 = "";
@@ -185,7 +185,7 @@ pub fn documentRange(doc: types.TextDocument, encoding: Encoding) !types.Range {
     var line_idx: i64 = 0;
     var curr_line: []const u8 = doc.text;
 
-    var split_iterator = std.mem.split(doc.text, "\n");
+    var split_iterator = std.mem.split(u8, doc.text, "\n");
     while (split_iterator.next()) |line| : (line_idx += 1) {
         curr_line = line;
     }

--- a/src/setup.zig
+++ b/src/setup.zig
@@ -225,7 +225,7 @@ pub fn findZig(allocator: *std.mem.Allocator) !?[]const u8 {
     const zig_exe = try std.fmt.allocPrint(allocator, "zig{s}", .{exe_extension});
     defer allocator.free(zig_exe);
 
-    var it = std.mem.tokenize(env_path, &[_]u8{std.fs.path.delimiter});
+    var it = std.mem.tokenize(u8, env_path, &[_]u8{std.fs.path.delimiter});
     while (it.next()) |path| {
         if (std.builtin.os.tag == .windows) {
             if (std.mem.indexOfScalar(u8, path, '/') != null) continue;

--- a/src/uri.zig
+++ b/src/uri.zig
@@ -57,7 +57,7 @@ pub fn pathRelative(allocator: *mem.Allocator, base: []const u8, rel: []const u8
     mem.copy(u8, result, base);
     var result_index: usize = base.len;
 
-    var it = mem.tokenize(rel, "/");
+    var it = mem.tokenize(u8, rel, "/");
     while (it.next()) |component| {
         if (mem.eql(u8, component, ".")) {
             continue;

